network-store-client/src/main/java/com/powsybl/network/store/client/PreloadingNetworkStoreClient.java

```diff
@@ -124,6 +138,33 @@ private void ensureCached(ResourceType resourceType, UUID networkUuid, int varia
         }
     }
 
+    private void ensureExtensionAttributesCached(ResourceType resourceType, UUID networkUuid, int variantNum, String extensionName) {
+        Objects.requireNonNull(resourceType);
+        Objects.requireNonNull(networkUuid);
+        Objects.requireNonNull(extensionName);
+        Map<ResourceType, Set<String>> extensionAttributesByTypeAndName = cachedExtensionAttributesByTypeAndName.getCollection(networkUuid, variantNum);
+        Set<ResourceType> extensionAttributesByType = cachedExtensionAttributesByType.getCollection(networkUuid, variantNum);
```

Comment:
rename extensionAttributesByType

Reply:
Simplified this in https://github.com/powsybl/powsybl-network-store/pull/411/commits/1d2c57e75f0193c6d358372294460f2872ac1d4d 
It was already handled in CollectionCache.